app/templates/events/view/tickets/attendees.hbs

```diff
@@ -0,0 +1,65 @@
+<div class="ui hidden divider"></div>
+<div class="ui compact menu pull right floated">
+  <div class="ui simple dropdown item">
+    {{t 'Showing all records'}}
+    <i class="dropdown icon"></i>
+    <div class="menu">
+      <div class="item">{{t 'Filter by Date'}}</div>
+      <div class="item">{{t 'Filter by Ticket'}}</div>
+      <div class="item">{{t 'Remove all Filters'}}</div>
+    </div>
+  </div>
+</div>
+<div class="ui compact menu pull right floated">
+  <div class="ui simple dropdown item">
+    {{t 'Export As'}}
+    <i class="dropdown icon"></i>
+    <div class="menu">
+      <div class="item">{{t 'PDF'}}</div>
+      <div class="item">{{t 'CSV'}}</div>
+    </div>
+  </div>
+</div>
+<div class="content">
+  <h2 class="ui header">{{t 'View Attendees'}}</h2>
+</div>
+<div class="ui hidden divider"></div>
+<div class= "content">
```

Comment:
the `content` class here, won't make any difference. Not required. Correct me if I'm wrong on this one :)